src/pages/_app.tsx

```diff
@@ -38,12 +37,7 @@ function App(props: AppProps) {
         />
       </Head>
 
-      <RootProvider
-        emotionCache={emotionCache}
-        defaultLocale={defaultLocale}
-        locale={locale}
-        {...other}
-      >
+      <RootProvider emotionCache={emotionCache} locale={locale} {...other}>
```

Comment:
So `defaultLocale` is no longer needed? :o

Reply:
Very good question.
We'll probably need it for monolingual apps?

Will try it out

Reply:
What do you mean? I can't see that `defaultLocale` has been re-added to this file. It needs to be added here for the `defaultLocale` set in `next.config.js` to be forwarded to the I18nProvider.